src/parser/transform/statement/transform_set.cpp

```diff
@@ -36,7 +38,7 @@ unique_ptr<SetStatement> Transformer::TransformSet(duckdb_libpgquery::PGNode *no
 		throw NotImplementedException("SET LOCAL is not implemented.");
 	}
 
-	auto name = std::string(stmt->name);
+	auto name = StringUtil::Lower(std::string(stmt->name));
```

Comment:
Rather than remembering to use `StringUtil::Lower` in all locations where SET variables are used, could we turn the `set_variables` into a `case_insensitive_map`?

Reply:
Oh wasn't aware of this. Thanks for the tip!